apps/watch/src/components/CollectionsPage/languages/en/CollectionsPage.tsx

```
@@ -0,0 +1,1378 @@
+              question: 'Why do I need saving if I’m a good person?',
+              answer: (
+                <>
+                  <p>
+                    The Bible says, “There is no one righteous, not even one”
+                    (Romans 3:10). God’s standard is perfection, and all have
+                    sinned and fall short of His glory (Romans 3:23). Being good
+                    by human standards isn’t enough to remove the guilt of sin.
+                    Salvation is not earned by good deeds but received by grace
+                    through faith in Jesus (Ephesians 2:8–9). Only His sacrifice
+                    can cleanse us and make us right with God.
+                  </p>
```

ESLint

[error] disallow literal string: <p>
                    The Bible says, “There is no one righteous, not even one”
                    (Romans 3:10). God’s standard is perfection, and all have
                    sinned and fall short of His glory (Romans 3:23). Being good
                    by human standards isn’t enough to remove the guilt of sin.
                    Salvation is not earned by good deeds but received by grace
                    through faith in Jesus (Ephesians 2:8–9). Only His sacrifice
                    can cleanse us and make us right with God.
                  </p>

(i18next/no-literal-string)